pages/index.js

```diff
@@ -16,67 +21,156 @@ export default function Home() {
+        About the Paste Prototyping Kit
+      </Heading>
       <Paragraph>
         The prototyping kit is a{' '}
         <PrototypeAnchor showExternal href="https://nextjs.org/">
           NextJS
         </PrototypeAnchor>{' '}
         app with Paste preloaded. It has everything you will need to create a fully functioning web based prototype
-        using Paste components
+        using Paste components.
       </Paragraph>
-      <Grid gutter="space40" marginBottom="space70" equalColumnHeights>
-        <Column>
-          <Card>
-            <Heading as="h3" variant="heading30">
-              Creating pages
-            </Heading>
-            <Paragraph>
-              Create pages in the <Code>pages</Code> directory. You can copy the <Code>template-page.js</Code> file and
-              rename it as <Code>my-new-page.js</Code> to get you going. The name of the file in the <Code>pages</Code>{' '}
-              directory will result in a route of the same name. e.g. <Code>my-new-page.js</Code> =&gt;{' '}
-              <Code>/my-new-page</Code>.
-            </Paragraph>
-          </Card>
-        </Column>
-        <Column>
-          <Card>
-            <Heading as="h3" variant="heading30">
-              Adding to the side nav
-            </Heading>
-            <Paragraph>
-              To add a link to a new page in the side bar, add the new page&apos;s file name to the pages.json file in
-              the <Code>pages</Code> directory. Make sure it is in quotes and followed by <Code>: []</Code>. For
-              example:
-            </Paragraph>
-            <Paragraph>
-              <Code>&quot;my-new-page&quot;: []</Code>
-            </Paragraph>
-            <Paragraph>
-              <strong>Note:</strong> if you are adding to the middle of the list, end the new line with a comma. If it
-              is the last list item, do not end the line with a comma.
-            </Paragraph>
-          </Card>
-        </Column>
-        <Column>
-          <Card>
-            <Heading as="h3" variant="heading30">
-              Linking between pages
-            </Heading>
-            <Paragraph>
-              <strong>Internal linking: </strong>Use the <Code>&lt;PrototypeAnchor /&gt;</Code> not the Paste Anchor. It
-              is a wrapper around the Paste Anchor suitable for NextJS websites. It looks and behaves exactly like a{' '}
-              <PrototypeAnchor href="https://paste.twilio.design/components/anchor" showExternal>
-                Paste Anchor
-              </PrototypeAnchor>
-              .
-            </Paragraph>
-          </Card>
-        </Column>
-      </Grid>
-      <Heading as="h2" variant="heading20">
-        Ways to get started
-      </Heading>
-      <Paragraph>The prototyping kit can be quickly deployed to the following platforms:</Paragraph>
+      <Paragraph>This prototype can be quickly shared by deploying to the following platforms:</Paragraph>
```

Comment:
I wonder if we should still provide more info about what the three different options are and how they differ?

Reply:
Mmmm yes yes. Do you know how you'll talk about them in the README? It could be very similar info here. We could collab on it?

Reply:
Yes! I'll ping you later about this